### PR TITLE
Update walkthrough for admin tasks

### PR DESCRIPTION
**Pull request by callunity**

### CODE REVIEW

File: 1-0-admin-tasks.md [hunk 1]
```diff
@@ -182,13 +182,81 @@ Now that your project has been created, continue to make your first samples.
 
 # 3. Creating groups and adding users to them
 
+_Groups_ permit fine-grained control over who has view and/or edit access to an
+item in MISO. There are default groups in MISO for "all internal users", as well
+as for RunWatchers, ProjectWatchers and PoolWatchers. In theory, members of a
+Watcher group will be notified by email of all events on members of that group
```
Comment (apmasell) [inline 1-0-admin-tasks.md]: Well, I learnt something today.

File: 1-0-admin-tasks.md [hunk 2]
```diff
@@ -182,13 +182,81 @@ Now that your project has been created, continue to make your first samples.
...
+   cannot contain spaces or punctuation.
+1. Enter a _Description_ of your group.
+1. Select your user as the only member of your group.
+1. Click the _Save_ button at the upper right.
+
+## 3.2 Update your project's permissions 
+
+Return to your new project's page and update the permissions:
+1. Click the _Permissions_ section divider to expand it.
+1. Click the _Allow all itnernal users access?_ box to deselect it.
```
Comment (apmasell) [inline 1-0-admin-tasks.md]: spelling internal

File: 1-0-admin-tasks.md [hunk 3]
```diff
@@ -182,13 +182,89 @@ Now that your project has been created, continue to make your first samples.
...
+sequencing machine's output folder to the notification server properties file.
+
+
 <a name="reports"/>
 
 # 5. Creating project and monthly reports
 
+MISO has some basic reporting capabilities. However, it also seems to contain
+dragons which periodically kill all of MISO when they are disturbed. Please
+avoid the _Reports_ tab so as not to anger the dragons.
```
Comment (djcooke) [inline 1-0-admin-tasks.md]: :fire:

Comment (callunity) [inline 1-0-admin-tasks.md]: Are you asking me to delete this? I'd prefer to keep it in, as we will get questions about reporting.

**Pull request merged**